packages/rocketchat-ui-account/account/accountProfile.html

```diff
@@ -59,6 +59,9 @@
 				<div class="submit">
 					<button class="button"><i class="icon-send"></i><span>{{_ "Save_changes"}}</span></button>
 				</div>
+				<div class="logoutOthers">
+					<button class="button">Logout Others</button>	
```

Comment:
@alayek you don't need to create a new PR, just change the files in your branch `logout-other-clients` commit and push then this PR will be updated.

Just for english, to add the new field to be translated to other languages at the LingoHup website.